flasc/data_processing/energy_ratio_wd_bias_estimation.py

```diff
@@ -158,7 +158,7 @@ def _load_a_input_for_wd_bias(
 
     def _get_energy_ratios_allbins(
         self,
-        wd_bias,
+        wd_bias: float,
```

Comment:
why type hint just this one?

Reply:
in `cost_fun`, the argument is a one-element numpy array; however, in `_get_energy_ratios_all_bins` there is some string formatting
```python
" WD bias: %.3f deg." % wd_bias
```
that raises a deprecation warning when `wd_bias` is a numpy array. This update indicates that `wd_bias` should be a float, not a one-element array. For consistency though, I could add type hints to the other arguments here.